### DISCUSSION TITLE
Change CSV Import Data Type

### DISCUSSION THREAD

**Discussion by duchuynh** [apache/superset#25614] [General]
Hello Team,

I imported a CSV File attached into a database:
[reports_sales_listings_transactions.csv](https://github.com/apache/superset/files/12872603/reports_sales_listings_transactions.csv)

Most of the columns that are supposed to be numeric for Superset to run effectively. What is the best way to change this datatype from the CSV file in a way that allows it to work?

**Comment by xavier-GitHub76** [comment]
Hello,

With the version 3.0.0, it is possible to set "Column Data Types"
#21906
